katharsis-core/src/main/java/io/katharsis/legacy/queryParams/QueryParams.java

```diff
@@ -61,6 +62,32 @@ void setIncludedRelations(TypedParams<IncludedRelationsParams> includedRelations
     }
 
     @Override
+    public int hashCode() {
```

Comment:
Where did you come up with this implementation of equals/hashcode ? It doesn't look like the intellij standard.